app/code/Magento/Catalog/Test/Mftf/Data/MsiCategoryData.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<entities xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:noNamespaceSchemaLocation="../../../../../../../dev/tests/acceptance/vendor/magento/magento2-functional-testing-framework/src/Magento/FunctionalTestingFramework/DataGenerator/etc/dataProfileSchema.xsd">
+    <entity name="Category1" type="category">
```

Comment:
Please do not create entities in Catalog module only for MSI.
You should use existing Categories or if you need a specific one, please extend `CategoryData.xml` from MSI module which dependent on Catalog module